social/tests/models.py

```diff
@@ -24,7 +24,7 @@ class User(BaseModel):
     cache = {}
     _is_active = True
 
-    def __init__(self, username, email=None):
+    def __init__(self, username, email=None, **extra_user_fields):
```

Comment:
Added this here to better mimic what Django allows during the create_user step and thus better test `USER_FIELDS`